magicctapipe/scripts/lst1_magic_real/lst1_magic_stereo_reco.py

```diff
@@ -162,75 +211,76 @@ def stereo_reco(input_data, output_data, config):
                 kurtosis=float(df_tel['kurtosis'].values[0]),
             )
 
-            event_array.dl1.tel[tel_id].parameters = ImageParametersContainer()
-
-        stereo_params = hillas_reconstructor._predict(
-            event_array,
-            hillas_params,
-            subarray,
-            array_pointing
-        )
+            event.dl1.tel[tel_id].parameters = ImageParametersContainer(hillas=hillas_params)
 
+        hillas_reconstructor(event)
+        stereo_params = event.dl2.stereo.geometry["HillasReconstructor"]
+            
         if stereo_params.az < 0:
-            stereo_params.az = stereo_params.az + u.Quantity(2*np.pi, u.rad)
+                stereo_params.az += u.Quantity(360, u.deg)
```

Comment:
here the indentation is wrong, should be one tab less

Reply:
thanks I modified it